canvas_modules/harness/cypress/e2e/canvas/links.cy.js

```diff
@@ -697,7 +707,7 @@ describe("Test selectedLinkSelection = 'Detachable' configuration option", funct
 		// First create a new node on the canvas.
 		cy.clickToolbarPaletteOpen();
 		cy.clickCategory("Record Ops");
-		cy.dragNodeToPosition("Sample", 300, 450);
```

Comment:
This doesn't look right. Looks like another instance where we need to click on the node before the drag.